cfgov/v1/models/browse_filterable_page.py

```diff
@@ -14,18 +15,29 @@
 from v1.util.filterable_list import FilterableListMixin
 
 
+class BrowseFilterableContent(StreamBlock):
+    """Defines the StreamField blocks for BrowseFilterablePage content.
+
+    Pages can have at most one filterable list.
+    """
+    full_width_text = organisms.FullWidthText()
+    filter_controls = organisms.FilterControls()
+    feedback = v1_blocks.Feedback()
+
+    class Meta:
+        block_counts = {
+            'filter_controls': {'max_num': 1},
+        }
+
+
 class BrowseFilterablePage(FilterableFeedPageMixin,
                            FilterableListMixin,
                            CFGOVPage):
     header = StreamField([
         ('text_introduction', molecules.TextIntroduction()),
         ('featured_content', organisms.FeaturedContent()),
     ])
-    content = StreamField([
-        ('full_width_text', organisms.FullWidthText()),
-        ('filter_controls', organisms.FilterControls()),
-        ('feedback', v1_blocks.Feedback()),
```

Comment:
Would it be possible to define the max inline here, rather than adding a whole new class?

Reply:
Unfortunately not the way the StreamField constructor works. There's no way to provided this meta information here -- that would be a nice improvement, though.